docs/SDL/GetListOfPermissions/index.md

```diff
@@ -10,75 +10,60 @@ Purpose
 
 !!! NOTE  
-a) The information about the application (name, ID, etc.) is provided by SDL via either [BasicCommunication.UpdateAppList](../../BasicCommunication/UpdateAppList/index.md) or [BasicCommunication.OnAppRegistered](../../BasicCommunication/OnAppRegistered/index.md) RPCs.  
-b) If HMI has never sent externalConsentStatus before (via [OnAppPermissionConsent](../OnAppPermissionConsent/index.md)), SDL will respond with empty array.  
-c) User can go to settings on HMI and enable or disable "External UCS" for an "entity". HMI will notify SDL accordingly (see figure ExternalConsentStatus).  
+a) The information about the application (name, ID, etc.) is provided by SDL via either BasicCommunication.UpdateAppList or BasicCommunication.OnAppRegistered RPCs.  
+b) If HMI has never sent externalConsentStatus before (via [OnAppPermissionConsent](https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/blob/master/docs/SDL/OnAppPermissionConsent/index.md)), SDL will respond with empty array.  
+c) User can go to settings on HMI and enable or disable "External UCS" for an "entity". HMI will notify SDL accordingly (_see figure ExternalConsentStatus_).  
 d) The resulting allowance of functional grouping is informed by parameter “allowedFunctions” to HMI in accordinance with result of ExternalConsentStatus.
-!!!
 
 #### Parameters
 
 |Name|Type|Mandatory|Additional|
 |:---|:---|:--------|:---------|
-|appID|Integer|false|If no app specified list of permissions for all apps|
+|appID|Integer|false||
 
 ### Response
 
 #### Parameters
 
 |Name|Type|Mandatory|Additional|
 |:---|:---|:--------|:---------|
-|allowedFunctions|[Common.PermissionItem]|true|array: true<br>minsize: 0<br>maxsize: 100|
-|ExternalConsentStatus|[Common.ExternalConsentStatus]|true|array: true<br>minsize: 0<br>maxsize: 100|
-[Common.PermissionItem]: ../../common/structs/#permissionitem
-[Common.ExternalConsentStatus]: ../../common/structs/#externalconsentstatus
-
-
+|allowedFunctions|[Common.PermissionItem](../../common/structs/#permissionitem)|true|array: true<br>minsize: 0<br>maxsize: 100|
+|ExternalConsentStatus|[Common.ExternalConsentStatus](../../common/structs/#ExternalConsentStatus)|true|array: true<br>minsize: 0<br>maxsize: 100|
 ### Sequence Diagrams
+
 |||
 GetListOfPermissions
 ![GetListOfPermissions](./assets/GetListOfPermissions.jpg)
 |||
 
-|||
 GetListOfPermissions without AppId
 ![GetListOfPermissions](./assets/GetListOfPermissionsNoId.jpg)
-|||
 
 |||
-GetListOfPermissions provide  External UCS Status to HMI   
+GetListOfPermissions provide  External UCS Status to HMI
 ![GetListOfPermissions](./assets/GetListOfPermissionsInformingHMI.png)
 |||
 
-|||
-Possible Layout - ExternalConsentStatus   
+Possible Layout - ExternalConsentStatus
 ![GetListOfPermissions](./assets/PossibleLayoutExternalConsentStatus.png)
-|||
-
 ### Example Request
 
 ```json
```

Comment:
Also, the example response (not in this diff) needs quite a bit of formatting work